### DISCUSSION TITLE
Tall and wide borders broken after last Windows 11 update

### DISCUSSION THREAD

**Discussion by tricbh** [Textualize/textual#4327] [Q&A] [answered]
Has anyone had a problem with the tall and wide borders after the last Windows 11 update? I have two stations with the same problem, which suggests it's not a problem specific to one computer. Python 3.11.8 and Textual 0.53.1

<img width="865" alt="2024-03-22_08-45_1" src="https://github.com/Textualize/textual/assets/155464595/544241a9-d8c7-469d-93bd-aadf78a1a1fc">

<img width="863" alt="2024-03-22_08-45" src="https://github.com/Textualize/textual/assets/155464595/d3c98b16-690c-49b2-a4f0-ef59d1552b68">

**Comment by davep** [comment]
What are the versions of Windows and Windows Terminal you're using there?

**Reply by tricbh** [reply]
Windows 11 Pro 23H2 with KB5035853 update. I am using cmd.exe to access command prompt and run "textual borders" in a specific Python enviroment (venv) with only Textual[-dev] installed (and its dependencies)

**Reply by davep** [reply]
Note Tom's advice below but, as [we say right at the start of the guide](https://textual.textualize.io/getting_started/), I would highly recommend using [Windows Terminal](https://apps.microsoft.com/detail/9n0dx20hk701?rtc=1&hl=en-gb&gl=GB) for running your application

**Reply by tricbh** [reply, marked as answer]
Same behavior with Power Shell 5.1, but it worked great with Power Shell 7. Thank you very much.

**Comment by TomJGooding** [comment]
Is your terminal font Cascadia? Apparently there's a known bug with how this unicode character used in these borders is rendered - see https://github.com/microsoft/terminal/issues/16265#issuecomment-1795918272 Try changing your terminal font?

**Reply by tricbh** [reply]
Worked with Power Shell 7. Thank you very much.
